brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1028 Fix: Typography: Line Height doesn't have decimal units in EM.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1028

### Screenshots

- https://d.pr/v/2tmwir

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

 - Drag and drop any block.
 - Go to typography > change font size and line-height unit to EM and increase the value.
 - It should increase in decimal like 1 to 1.1 in this way.
 - By changing the unit to PX it will increase like 1 to 2 in this way.

### Checklist:
